js/pro/bybit.js

```diff
@@ -1151,6 +1152,7 @@ module.exports = class bybit extends bybitRest {
             'remaining': undefined,
             'status': status,
             'fee': fee,
+            'trades': [ this.safeString (order, 't') ],
```

Comment:
@pcriadoperez This is not the expected type, you should populate it with `Trades` not strings